### DISCUSSION TITLE
Introductions: Say hi!

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#14] [General]
Welcome to RAG Hack. Please use this thread to say hi!

Suggested details:

📛 name/nickname/username
🌎 location (country and state/province if relevant)
🤖 previous AI experience
😻 fav hobby outside coding

**Comment by multispark** [comment]
I'll go first!

📛 Josh Oh
🌎 Seattle, Washington
🤖 I am a PM working with the Python DevRel team at Microsoft. I do a lot of work related to LLMs and helped organize the [predecessor hack](https://github.com/microsoft/AI-Chat-App-Hack) to RAG Hack.
😻 Some of my fav hobbies include playing musical instruments 🎸🎷, participating in snow sports 🎿, and traveling!

**Reply by yusciti** [reply]
Am Yusuf Taofik, Am very happy to be here
 Am working with Abuja Electricity, I love working with LLM and I am here to learn from professionals.
I love watching football and reading news

**Reply by snlinfo2** [reply]
You are welcome Yusuf

**Comment by pamelafox** [comment]
📛 Pamela Fox
🌎 Bay Area, California
🤖 I am a Python Cloud Advocate, and I love making repos that show how to combine Python + Microsoft products. Check out my GitHub profile for lots of them!
😻 Besides raising my two lil kids, I also like to grow native pollinator-friendly plants and watch caterpillars pupate on them! 🐛

**Comment by saifkabbo** [comment]
📛 Abu Saif
🌎 Bangladesh
🤖 I am a Python developer.
😻Some of my favorite hobbies include playing card , reading book, cooking and traveling!

**Comment by sam-dude** [comment]
📛 Samuel Ibiyemi
🌎 Nigeria
🤖 I am a Python developer. I also build client-side applications with JavaScript.
😻 I enjoy reading research papers and meeting new people.

**Reply by Yash-Kavaiya** [reply]
Welcome @sam-dude nice to meet you

**Comment by thesumedh** [comment]
📛 Sumedh 
🌎 India
🤖 I am a Python developer. 
😻 I enjoy reading books and working out.

**Reply by JeffersonPires3** [reply]
I am From Brasil 🇧🇷
I am a Python developer. I also build client-side applications with JavaScript.

**Comment by navid-ahrary** [comment]
📛 Navid Ahrary
🌎 Iran
🤖 I am Azure Cloud Developer using Node.js, Python, and Azure-services. Certified Az-900 and AI-900. Working on AZ-204 nowadays. I actively collaborate with Microsoft communities.  Please check my professional wall on [Linkedin](https://www.linkedin.com/in/navid-ahrary-956a94193/)
😻 Interested in soccer, cross-fit, hangout with friends, travel, reading philosophy books.

**Comment by sujith-rkumar** [comment]
Will the recording of the sessions be available? @multispark

**Reply by HasnainSolangi** [reply]
Yes! here it is.
https://www.youtube.com/watch?v=f6pUqDeMiG0&list=PLmsFUfdnGr3y8rogpVg80cZOxhYeuoS2j&index=4

**Comment by HasnainSolangi** [comment]
📛 Hasnain Ahmed
🌎 Karachi, Sindh, Pakistan
🤖 AI Experience: I am currently diving into Generative AI, with the recent completion of the Microfit Student Ambassadors program, where I explored cloud computing, Azure AI, and their foundational applications. An experience I found truly amazing!
😻 Favorite Hobby: Reading books, articles, tech trends, staying updated on current news, and enjoying cricket.

**Comment by osama-swe** [comment]
📛 Osama Ayman (https://www.linkedin.com/in/osama-ayman/)
🌎 Cairo, Egypt (GMT+3)
🤖 AWS Certified Cloud Practitioner, Google Cloud - Prompt Design in Vertex AI Skill Badge, and Nvidia Jetson AI Lab Research Group attender.
😻 Fins-Swimming & Horse Riding.

**Comment by snlinfo2** [comment]
> Welcome to RAG Hack. Please use this thread to say hi!
> 
> Suggested details:
> 
> 📛 name/nickname/username 🌎 location (country and state/province if relevant) 🤖 previous AI experience 😻 fav hobby outside coding

Ajayi6, Abuja, Nigeria, I am a beginner. I am glad to join the RAG hack!

**Reply by 4TripleA** [reply]
Hafa Ajayi6, let's team up bro

**Comment by 4TripleA** [comment]
Abdurrahman, Nigeria, Am into machine learning and diving into deep learning, coding

**Comment by sam-0706** [comment]
hi guys
sam here
GEN-AI data-scientist here,
from INDIA

**Reply by Yash-Kavaiya** [reply]
Welcome @sam-0706 City ?

**Comment by 4TripleA** [comment]
Hi, can we team up bro

**Reply by sam-0706** [reply]
Sure!

**Comment by roxStarRob** [comment]
# 📛 **RoB** - he/him/his and us/ours/we;

- 🌎 Seattle, WA area (currently);
- 🤖 familiar _beginner_;
- 👨🏾‍💻 JavaScript/TypeScript primarily (FE), Vue, Node, C#, OOP, FP, etc... (I believe we can figure out whatever);
> 😻 Listening to music, watching movies, and eating professionally skillfully prepared cultural food;

**Comment by chin7an** [comment]
hi all!
📛 Chintan
🌎 Seattle
🤖 not much AI experience outside a couple of work hackathon projects
😻 currently exploring woodworking with hand tools, like in the old days!

**Comment by rahulmansharamani14** [comment]
📛 Rahul Mansharamani
🌎 Chicago, Illinois
🤖 CS grad student from IIT Chicago. I spent my summer at Motorola Solutions working on RAG-based LLM application. Won an internal company hackathon also RAG based. I love to solve problems using LLMs.
😻 Documentaries, Reading tech articles, premier league soccer (Go Spurs!)

**Comment by Manikant92** [comment]
Hello everyone!

**Comment by vikram-cloud-ai** [comment]
📛 vikram
 🌎 bengaluru, india
 🤖 working on learning and implementing pocs on creating applications using azure open ai , python 
 😻 Watching cricket, movies

**Reply by Yash-Kavaiya** [reply]
Welcome @vikramarigelaRepo

**Comment by 4TripleA** [comment]
Give me your Whatsapp contact, let's discuss

**Comment by harshal-14** [comment]
📛 harshal-14
🌎 USA
🤖 Machine Learning Intern and Robotics Engineer
😻 Travel
Looking for teammates for hackathon

**Reply by Yash-Kavaiya** [reply]
@harshal-14 I participated in many RAG based hackathon Let's create a team

**Comment by mariogarcia-ar** [comment]
📛 mariogarcia_ar
🌎 Rosario - Santa Fe - Argentina
🤖 MGc Data Scientist / Micromaster Statistics and Data Science
😻 i+d+i / programming

**Comment by divyabhutani** [comment]
📛 Divya Bhutani
🌎 Chicago, Illinois
🤖 I'm a Computer Science graduate student from the University of Florida and I actively participate in tech events and hackathons which helped me in securing first position at MacHacks(North America's first AI focused hackathon).
😻 I love traveling and reading books.

**Comment by codingkush** [comment]
📛Kush Gupta
🌎India, Chhattisgarh
🤖 Basic understanding of AI concepts, working on integrating AI for responsive systems
😻 Reading books, exploring mythical philosophies and their intriguing tales.

**Comment by icodebest** [comment]
📛 Waleed Usman
🌎 Pakistan
🤖 Machine Learning Engineer, currently learning about LLM and RAG applications
😻 Reading books, music, exploring the beauty of nature

**Comment by CarlOMattsson** [comment]
📛 Carl O Mattsson 
🌎 Sweden, Växjö 
🤖 I'm starting to know what I don't know
😻 Loves tabletop role playing games (like D&D and others)

**Comment by Mr-Nnobody** [comment]
Hello everyone,

📛 Lewis
🌍Cameroon
🤖I am a python developer and an AI novice
😍 Gaming and I play instruments 🥁 🎹

**Comment by Rustem** [comment]
📛 Rustem
🌍United States / Kazakhstan
🤖I am an engineer and an AI novice
😍 Martial arts and reading books

**Comment by faizan-yousaf** [comment]
📛 Muhammad Faizan
🌎 Pakistan 
🤖 I am a Python developer, and an enthusiastic Data Scientist.
😻Some of my favorite hobbies include Photography, reading books, and traveling!

**Comment by JasonHaley** [comment]
📛 Jason Haley
🌎 Boston area
🤖 I am a full stack developer using mostly Microsoft technologies. Here to learn from others and the presentations.
😻 User group leader, podcaster, ultra runner and home coffee roaster

**Comment by vikram-cloud-ai** [comment]
Hey can we form a team ?

Regards,
vikram

**Comment by snlinfo2** [comment]
Yes, but I will have to be your lead team member, if you can confirm that them, we are good to go!
Regards,
Oluwafemi Temitope Victor Ajayi ( Just call me "Ajayi6")

________________________________

**Reply by harshal-14** [reply]
Can I also join your team? I'd like to learn a lot from you guys

**Reply by snlinfo2** [reply]
Yes, send me consent email to the email below. Thanks

**Comment by snlinfo2** [comment]
Hi, Guys,
We are trying to form a team, if your interested, kindly send me your written consent email.
***@***.***
Thanks
Regards,
O. T. V. Ajayi (Simply call me "Ajayi6")

________________________________

**Reply by snlinfo2** [reply]
My info2@speednetapps.com 
For the consent to join my team

**Comment by temesgen5335** [comment]
📛 Temesgen Gebreabzgi
🌎 Addis Ababa, Ethiopia
🤖 I'm a Junior AI Engineer, skilled in Machine Learning and Data Engineering. Currently working as a junior AI Engineer at 10academy.org 
Besides work i spend most of my time playing online Chess. if you dare to challenge me am at Chess.com with the name: TheUnderdogPeasant

**Comment by henockk** [comment]
📛 Henock Dessalegn
🌎 location Addis Ababa, Ethiopia
🤖 I am currently a Generative AI Intern at 10 Academy
😻 I love movies

**Comment by patroswastik** [comment]
📛 Swastik Patro
🌎 Chicago, Illinois
🤖 I'm a master's student at Illinois Institute of Technology, Chicago. Currently majoring in Computer Science. And on a journey of unraveling machine learning mysteries, fueled by enthusiasm and a love for algorithms.
😻 Favorite Hobby: Movies, Sports, Video Games and (I am Batman! 🦇)

**Reply by snlinfo2** [reply]
Patro are you in any team for Hack Together or you are doing solo?

**Comment by lumensparkxy** [comment]
📛 maswadkar
🌎 Switzerland
🤖 trying to explore AI world and real life application for education.
😻 trying to teach chess to my 8 year old

**Comment by learnsol** [comment]
📛 Siva
🌍New Jersey, USA 
🤖I am an database engineer and an AI learner
😍 Music and gaming

**Comment by Yash-Kavaiya** [comment]
Hello 

📛 Yash-Kavaiya
🌎 Pune , India
```bash
www.linkedin.com/in/yashkavaiya
```
😻 Traveling, Cricket

**Comment by aloha-fim** [comment]
📛 Frederec Im
🌎 Canton Zug
🤖 I like Python and Postgres, in that order.
😻 I moved to Switzerland because my wife made me.

**Comment by WajahatAliHassan** [comment]
📛 Wajahat Ali Hassan (WajahatAliHassan)
🌎 Pakistan
🤖 I am a Full Stack Developer (JavaScript and Python) with strong background in AI and web 
😻 Teaching, Reading Books

**Comment by mahmoodayesha** [comment]
📛 Ayesha 
🌎 Seattle, WA
🤖 I am a Full Stack Developer with a strong background in , Python , AI,   MERN, and MEAN stack.
😻 Love Reading Books
I recently discovered about this hackathon and looking to form a team, my discord user  is @ysh200310

**Comment by IamRam3** [comment]
📛 ram
🌎 Hyderabad, India
🤖 Recent graduate, specifically looking for AI roles.
😻 Playing chess, BGMI and listening music

**Comment by DevMarkson** [comment]
Favour Markson. 
Lagos, Nigeria
I have worked on computer vision, OCR and touch the basics of NLP
I'm an avid chess player.
